tests/data/rolling_stocks/short_slow_rolling_stock.json

```diff
@@ -76,5 +76,6 @@
     }
   },
   "base_power_class": "1",
-  "energy_sources": []
+  "energy_sources": [],
+  "supported_signaling_systems": ["BAL", "BAPR"]
```

Comment:
I don't think these should differ from the rest, the different version of the generic rolling stock generally only differ in what's described in their name. You could add a new rolling stock if it's relevant. 

(same for the slow_rolling_stock)

Reply:
resolved in previous PR